frontend/ui/widget/touchmenu.lua

```diff
@@ -594,16 +594,26 @@ function TouchMenu:updateItems()
     table.insert(self.item_group, self.footer_top_margin)
     table.insert(self.item_group, self.footer)
     -- @translators %1 is the current page. %2 is the total number of pages. In some languages a good translation might need to reverse this order, for instance: "Total %2, page %1".
-    self.page_info_text.text = util.template(_("Page %1 of %2"), self.page, self.page_num)
+    -- Don't show page x of x if there is only one page.
+    if self.page_num > 1 then
+        self.page_info_text:setText(util.template(_("Page %1 of %2"), self.page, self.page_num))
+    else
+        self.page_info_text:setText("")
+    end
     self.page_info_left_chev:showHide(self.page_num > 1)
     self.page_info_right_chev:showHide(self.page_num > 1)
     self.page_info_left_chev:enableDisable(self.page > 1)
     self.page_info_right_chev:enableDisable(self.page < self.page_num)
-    local time_info_txt = os.date("%H:%M").." @ "
-    if Device:getPowerDevice():isCharging() then
-        time_info_txt = time_info_txt.."+"
+
+    local time_info_txt = os.date("%H:%M").." – "
+    local twelve_hr = G_reader_settings:isTrue("twelve_hour_clock")
+    if twelve_hr then
```

Comment:
`if G_reader_settings:isTrue("twelve_hour_clock") then` is ok when it's that short.

Reply:
And make that a `if else` to avoid 2 calls to `os.date()`:
```lua
local time_info_txt
if G_reader_settings:isTrue("twelve_hour_clock") then
  time_info_txt = os.date(...)
else
  time_info_txt = os.date(...)
end
time_info_txt = time_info_txt .." – " .. Device:getPowerDevice():getCapacity().."%"
```